app/views/admin/buildings/_flash_content.html.erb

```diff
@@ -0,0 +1,27 @@
+<% flash.each do |name, msg| %>
+  <% if msg.include?("not updated") %>
+    <div class="govuk-error-summary govuk-!-margin-top-6 govuk-!-margin-bottom-6" aria-labelledby="error-summary-title" role="alert" tabindex="-1">
+      <svg class="alert__icon" fill="red" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg"
+           viewBox="0 0 25 25" height="25" width="25">
+        <path d="M13.6,15.4h-2.3v-4.5h2.3V15.4z M13.6,19.8h-2.3v-2.2h2.3V19.8z M0,23.2h25L12.5,2L0,23.2z"></path>
+      </svg>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <%= msg %>
+          </li>
+        </ul>
+      </div>
+    </div>
+  <% else %>
+    <div class="govuk-!-margin-top-6 govuk-!-margin-bottom-8">
+      <div class="govuk-error-summary__body govuk-!-padding-2">
+        <svg class="moj-banner__icon" color="green" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg"
+             viewBox="0 0 25 25" height="25" width="25">
+          <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z"></path>
+        </svg>
+        <%= msg %>
```

Comment:
What's special about the "not updated" error? I can't see the difference besides a bullet point.

This is the flash on the side of the form:

```erb
<% if @survey.errors.any? %>
  <div class="govuk-error-summary">
    <h2 class="govuk-error-summary__title">
      There is a problem
    </h2>
    <div class="govuk-error-summary__body">
      <ul class="govuk-list">
        <% @survey.errors.full_messages.each do |message| %>
          <li>
            <span class="govuk-error-message"><%= message %></span>
          </li>
        <% end %>
      </ul>
    </div>
  </div>
<% end %>
```

Reply:
I'll defer this one until after the other bulk actions are implemented as we might need something else entirely.